### DISCUSSION TITLE
Encounter a problem Upload proof of your academic status

### DISCUSSION THREAD

**Discussion by Thirdvespa** [community/community#56377] [GitHub Education]
### Hi everyone,

I upload an img student card  but  github show "Your application cannot be reviewed until you fix the following:
You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.
Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy." 

I solved it but still not. What should I do? help me pls.

**Comment by plebgang** [comment]
Yes,that's right

**Comment by samus-aran** [comment]
Hi @Thirdvespa 👋 

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Let us know how you get on 👍
